packages/grid/package.json

```diff
@@ -0,0 +1,13 @@
+{
+ "name": "@starryui/grid",
+ "version": "1.0.0",
```

Comment:
Version is wrong, should match all other components

Reply:
Done

Reply:
✅ Perfect